src/globalStyles/global.scss

```diff
@@ -359,6 +359,7 @@ th.reactable-header-sort-asc:after {
 
 #mainColumn {
   padding-top: $gutterWidth;
+  width: 100%;
```

Comment:
what does this solve?  need to be careful changing template.

Reply:
This solves the mainColumn to exceed the header that contains text `ueried genes are altered in 764...` extending off-screen (see screenshot above). I have updated the scope of this change somewhat and used max-width to be more careful:
```
.contentWidth {
   #mainColumn {
    ...
    max-width: 100%;
  }
```